manifests/s/SparkLabs/openvpn-configuration-generator/1.0.8.1008/SparkLabs.openvpn-configuration-generator.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: SparkLabs.openvpn-configuration-generator
+PackageVersion: 1.0.8.1008
+PackageLocale: en-US
+Publisher: SparkLabs Pty Ltd
+PublisherUrl: https://www.sparklabs.com
+PublisherSupportUrl: https://www.sparklabs.com/support
+PrivacyUrl: https://www.sparklabs.com/company/legal/privacy
```

```suggestion
PrivacyUrl: https://www.sparklabs.com/company/legal/#privacy
```
This is 100% my fault, my YamlCreate script removes all `#` 🤦‍♂️ and i don't really know how to work around it

---

Oke.. Solved it🤦‍♂️
`$Variable = $Line.Replace("#","").Split(":").Trim()` -> `$Variable = $Line.TrimStart("#").Split(":").Trim()`
imma try to get some more changes in

---

That's nice!

---

#20359 actually fixes more issues than expected 🎉